AT/dialogue/command_types/Detection/1000.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
needs an "objective" to work with new dialogue system, same applies to all JSON files, examples can be seen in EOSS json files